package.json

```diff
@@ -7,12 +7,15 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^1.1.0",
```

Comment:
- [ ] OPTIONAL
Kindly endeavor to remove unused package to keep your external modules compact.